lisa/tests/kernel/test_bundle.py

```diff
@@ -85,6 +86,11 @@ def lower_name(self):
         """Return the name in lower case"""
         return self.name.lower()
 
+class ProcessResultBundle:
```

Comment:
That is exclusively used by Exekall, as such I'd like it not to be visible to LISA. Having it as a "private" class would hide it from autocomplete, but I would rather have the code not live here. Could this live in the glue instead?